### ISSUE TITLE
Support importing assertions along with the store

### ISSUE THREAD

**Issue by poovamraj** [openfga/cli#224]
Currently we support importing a store file using

`fga import store --file=store.fga.yaml`

But this doesn't import the assertions present along with the store.

Importing the assertions can be helpful to be accessed from the playground and do the checks.

**Comment by rhamzeh** [comment]
Blocked by:
- [ ] https://github.com/openfga/openfga/issues/946
- [ ] https://github.com/openfga/openfga/issues/1353

**Comment by KamalAman** [comment]
I would also love a `fga model test --upload` command to upload the assertions into the playground

**Comment by sujitha-av** [comment]
Hey all, when we are talking about importing assertions, it is only the `check assertions` right? since that is the only type that can be created/accessed from the playground.

**Comment by aaguiarz** [comment]
Yes, you'd convert check tests to assertions.

**Comment by sujitha-av** [comment]
> Yes, you'd convert check tests to assertions.

Yes, but then there are other assertion types too like the `listObjects` and `listUsers` [assertions](https://openfga.dev/docs/modeling/testing#write-list-objects-tests).  Do we import them too
1. They are not supported in the playground
2. The `WriteAssertions` [API](https://pkg.go.dev/github.com/openfga/go-sdk@v0.6.3#OpenFgaApi.WriteAssertions) only supports check assertions. Should we do this first and support the other types of assertions too?

**Comment by aaguiarz** [comment]
No, they should be ignored for now. We should only import assertions for Check

**Comment by sujitha-av** [comment]
Hello, I need some help with this: 
Was trying to test the import function. Realised that we create another `fgaClient` midway here in `CreateStoreWithModel` -  https://github.com/openfga/cli/blob/6c3b7aae75a3c81ddaec8abf70b38c2901b33531/cmd/store/create.go#L50-L59
I tried to refactor it here - https://github.com/sujitha-av/cli/commit/df157a1721cbef004f900b4bbe84f82d434a3780
But `CreateStoreWithModel` requires a `*client.OpenFgaClient` due to this line https://github.com/openfga/cli/blob/6c3b7aae75a3c81ddaec8abf70b38c2901b33531/cmd/store/create.go#L74
The problem is there is a mock for client of type `client.SdkClient` but not for `OpenFgaClient` which implements it and has extra methods like `SetStoreId`.  Any help on how to proceed is appreciated, thanks!

**Comment by ewanharris** [comment]
@sujitha-av I just opened [a PR](https://github.com/openfga/go-sdk/pull/155) in the go-sdk to add the getter/setter for Store ID to the `SdkClient` interface so that the generated mock will then include that method. I'll ask the team to review this to make sure it's correct.

We need to maintain the `SetStoreId` method usage because when we create a store we need to ensure that the `OpenFGAClient` is updated to use the ID from the store created in `CreateStoreWithModel` in subsequent API calls.

**Comment by sujitha-av** [comment]
Sure, thank you!

**Comment by ewanharris** [comment]
@sujitha-av FYI we just merged https://github.com/openfga/cli/pull/436 to `main` that updates the go-sdk to include that change to the interface and generate the mocks. If you pull and update your branch that mock should now the `SetStoreId` method.

**Comment by ewanharris** [comment]
Hey @sujitha-av, I just wanted to check in and see if there's any assistance we can provide. If the work is complete and you're just needing to add tests, feel free to open a PR as the testing in this project isn't necessarily the most straight forward due to mocking etc.

**Comment by sujitha-av** [comment]
Thanks @ewanharris. Yep, got carried away in testing other parts of import. I have opened a PR for just this now.
